### PR TITLE
go version updated to 1.21

### PR DESCRIPTION
**Pull request by nvvfedorov**
Dockerfiles were updated to use GO1.21.

How to test:

1. Build images
```
make local
```
3. Run ubuntu20.04

```
docker run --gpus all --rm -p 9400:9400 docker.io/nvidia/dcgm-exporter:3.3.0-3.2.0-ubuntu20.04
```

4. After ~30 seconds, make an HTTP request:
```
curl -v localhost:9400/metrics
```

You expected to see a list of metrics.

5. Run Dockerfile.ubi8

```
docker run --gpus all --rm -p 9400:9400 docker.io/nvidia/dcgm-exporter:3.3.0-3.2.0-ubi8
```

6. After ~30 seconds, make an HTTP request:
```
curl -v localhost:9400/metrics
```

You expected to see a list of metrics.

### CODE REVIEW

File: docker/Dockerfile.ubi8 [hunk 1]
```diff
@@ -1,6 +1,34 @@
...
+    #     ;; \
+    # *) echo >&2 "error: unsupported architecture '${arch}'"; exit 1 ;; \
+    # esac; \
+    # \
+    # wget -O go.tgz "$url" --progress=dot:giga; \
+    # # https://github.com/golang/go/issues/14739#issuecomment-324767697
+    # tar -C /usr/local -xzf go.tgz; \
+    # rm -f go.tgz*; \
+    # \
+    # go version
```
Comment (nikkon-dev) [inline docker/Dockerfile.ubi8:31]: Are these commented lines still needed?

Comment (vfedoroff) [reply docker/Dockerfile.ubi8:31]: My bad. Thanks for your findings.

**Review by nikkon-dev — APPROVED** [review]
LGTM

**Review by glowkey — DISMISSED** [review]

**Review by nikkon-dev — DISMISSED** [review]

**Review by nikkon-dev — APPROVED** [review]

**Comment by nvvfedorov** [comment]
Vulnerability report received:
		 0 Critical
		 0 High
		 4 Medium
		 27 Low
		 3 Negligible
		 0 Unknown

**Review by glowkey — APPROVED** [review]

**Pull request merged**